src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,19 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieTypes } from '../../types/MovieTypes';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movie: MovieTypes[];
+};
+export const MoviesList: React.FC<Props> = ({ movie }) => (
+  <>
+    <div className="movies">
+      {movie.map(movieEl => (
```

Comment:
And that would enable you to use singular form of the noun here, where you are actually iterating over each movie in the array
```suggestion
      {movies.map(movie=> (
```

Reply:
movie.map - you blew my mind